skypy/halos/sampler.py

```diff
@@ -0,0 +1,59 @@
+'''Halo mass sampler.
+This code samples cluster halos from colossus mass function .
+
+'''
+import numpy as np
+import colossus as colossus
+from colossus.lss import mass_function
+from scipy import integrate
+
+def colossus_mass_sampler(redshift, model, mdef, m_min, m_max, cosmology, sigma8, ns, size=None, resolution=1000):
+    """
+        This function generate a sample of cluster with halo mass from Despali+16 mass function.
+       
+    Parameters
+    -----------
+    redshift: float
+     Redshift
+    model: string
+     Mass function model which is available in colossus
+    mdef: str
+     halo mass definition, any spherical overdensity. For example: '200m', 200c', '500c', '2500c'
+         ***If select=True, then mdef has to be '500c'.
+    m_min, m_max: float
+     Lower and upper bounds for cluster mass (Unit: Msun).
+    cosmology: astropy.cosmology.Cosmology
```

Comment:
I would just say "Astropy cosmology object"; they will always be valid since they all require H0 and Om0 by definition, but Ob0 is not strictly necessary. Also `omega_matter` and `omega_matter` don't match the variable names used by astropy so could be confusing.